components/intro/intro.module.scss

```diff
@@ -0,0 +1,18 @@
+@import '../../styles/colors';
+
+.home {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.logo {
+  margin-bottom: 30px;
+  width: 250px;
+}
+
+.version {
+  color: #123837;
```

Comment:
Add this to the colors scss variables file.

Reply:
Listo